HW2/task2.py

```diff
@@ -0,0 +1,24 @@
+# Напишите программу, которая получает целое число и возвращает его
+# шестнадцатеричное строковое представление.
+# Функцию hex используйте для проверки своего результата.
+
+def decimal_to_hexadecimal(n):
+    if n == 0:
+        return '0'
+
+    hex_digits = "0123456789ABCDEF"
+    hex_string = ''
+
+    while n > 0:
+        remainder = n % 16
+        hex_string = hex_digits[remainder] + hex_string
```

Comment:
Если переменная используется единожды, то ее лучше не определять 
```suggestion
        hex_string = hex_digits[n % 16] + hex_string
```